scripts/tests/sanitycheck/test_testsuite_class.py

```diff
@@ -3,17 +3,20 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 # pylint: disable=line-too-long
-# pylint: disable=C0321
+# pylint: disable=multiple-statements
```

Comment:
the whole idea behind linting is that you follow them, if for some reason you have to disable something, you should add a comment about why this rule does not apply to your change...